src/index.js

```diff
@@ -1,3 +1,137 @@
+// 전역 변수
+const todoItem = `
+<div class="view">
+  <input class="toggle" type="checkbox"/>
+  <label class="label">새로운 타이틀</label>
+  <button class="destroy"></button>
+</div>
+<input class="edit" value="새로운 타이틀" />
+`;
+let state = {
+    condition: "all",
+    items: [],
+};
+const todoList = document.getElementById("todo-list");
+
+// state객체의 상태에 따라 새로고침 해주는 메소드
+// state.condition 에 따라 각각 다른 리스트를 보여준다.
+// 함수의 마지막에 localStorage에 저장하는 코드가 있음
+function rePaint(state) {
+    todoList.innerHTML = '';
+    let rePaintList = [];
+    if (state.condition === "all") rePaintList = state.items; 
+    if (state.condition === "active") rePaintList = state.items.filter(item=> item.completed === false);
+    if (state.condition === "completed") rePaintList = state.items.filter(item=> item.completed === true);
```

Comment:
```suggestion
  if (state.condition === "active") rePaintList = state.items.filter(item=> !item.completed);
  if (state.condition === "completed") rePaintList = state.items.filter(item=> item.completed);
```
다음과 같은 방법은 준혁님이 작성해주신 것보다는 덜 명시적이지만, 
completed 값이 boolean 이라면 다음과 같이 수정할 수 있습니다.